### ISSUE TITLE
Consume GTFS Realtime data from Foothill Transit

### ISSUE THREAD

**Issue by evansiroky** [cal-itp/data-infra#1084]
The Foothill Transit agency has GTFS-RT feeds that explicitly require requests to originate from a specific static IP address in order to be authenticated (see [Foothill Transit's Developer Resources page](http://foothilltransit.org/about/developer-resources/)). The gtfs-rt-archiver script or some of the data pipeline infrastructure should be able to make requests from a static IP address in order to fulfill this requirement.

**Comment by e-lo** [comment]
Question: is this a reasonable request for a technical implementer? Wondering if we need to add this to GTFS Guidelines 3.0

**Comment by evansiroky** [comment]
I don't think it is. It seems to be a way to do authentication, but that can be done with API keys which is much less difficult for end-users than setting up custom infrastructure to ensure requests come from a static IP.

**Comment by atvaccaro** [comment]
@evansiroky is this something we can push into the icebox? I don't think this is trivially accomplished within our current Google Cloud setup, but I also don't know the urgency of having Foothill Transit RT data.

**Comment by holly-g** [comment]
See Slack thread for discussion: https://cal-itp.slack.com/archives/C014Q6G3VCJ/p1645635601688579 

Decision was made to icebox this issue and follow up with Foothill Transit after we publish GTFS Guidelines 2.1
